deploy/operator.yaml

```diff
@@ -18,7 +18,7 @@ spec:
         - name: jaeger-operator
           image: jaegertracing/jaeger-operator:1.10.0
           ports:
-          - containerPort: 60000
+          - containerPort: 8383
```

Comment:
Does this port have any particular significance? Would it be better to have something in the same ballpark as other Jaeger ports?

Reply:
The port 60000 didn't seem to be used and is probably a left-over from a copy/paste or generated code.

The port 8383 is the one used in newly created operators.